pkg/sql/lease.go

```diff
@@ -795,6 +795,7 @@ func (t *tableState) upsertLocked(ctx context.Context, table *tableVersionState,
 	s := t.mu.active.find(table.Version)
 	if s == nil {
 		t.mu.active.insert(table)
+		log.Eventf(ctx, "new lease: %s with %s", s, table)
```

Comment:
Came across this while playing with gccgo - doesn't this NPE every time since `s` is nil?

Reply:
Good point @tamird 

```
 [client=127.0.0.1:49864,user=root,n1] new lease: <nil> with 51("kv") ver=2:1517397883.950948253,0, refcount=0
```

looks like it logs a nil rather than call the String() method on the nil pointer. I'll fix it.

Reply:
https://github.com/cockroachdb/cockroach/pull/22243